es6.js

```diff
@@ -0,0 +1,59 @@
+class BuilderES6 {
+    constructor(integer) {
+        this.integer = integer || 0;
+    }
+}
+
+class IntBuilder extends BuilderES6 {
+    plus(...n) { 
+        this.integer = [...arguments, this.integer].reduce((accum, curr) => accum + curr, 0);
+        
+        return this.integer;
```

Comment:
this only returns the value, in order to make the method chain you need to return the object itself. Hint: It's a special keyword in JS.